app/assets/javascripts/ideas.js

```diff
@@ -81,7 +81,8 @@ var Ideas = {
 
     delete: function(event) {
         var id = $(event.target).parent('li').data('id');
-        $.ajax('/ideas/'+id, { method: 'DELETE' }).success(function() { Ideas.loadAll(); })
+        $.ajax('/ideas/'+id, { method: 'DELETE' })
+            .success(function() { Ideas.loadAll(); })
```

Comment:
Added a line break here to make the start of the promise clearer and to be more consistent with the rest of the code.